test/parallel/test-repl-null.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+require('../common');
+const repl = require('repl');
+const assert = require('assert');
+
+var replserver = new repl.REPLServer();
+
+replserver._inTemplateLiteral = true;
```

Comment:
Oh, and I was testing this particular corner case because I wanted to make sure that my refactoring the code didn't alter the behavior. And this seemed to me like odd behavior for which we had no tests.